Content.Server/Flash/FlashSystem.cs

```diff
@@ -132,13 +132,18 @@ public void Flash(EntityUid target, EntityUid? user, EntityUid? used, float flas
             flashable.LastFlash = _gameTiming.CurTime;
             flashable.Duration = flashDuration / 1000f; // TODO: Make this sane...
             Dirty(flashable);
+                
+            if (user is not null)
+            {
+                RaiseLocalEvent(target, new FlashEvent(target, user.Value));
+            }
```

Comment:
What about stuff like portable flashers where there is no user, it should probably still raise the event.